cmd/state-custom-installer/runUnInstall.go

```diff
@@ -0,0 +1,61 @@
+package main
+
+import (
+	"fmt"
+	"path/filepath"
+
+	"github.com/ActiveState/cli/internal/config"
+	"github.com/ActiveState/cli/internal/errs"
+	"github.com/ActiveState/cli/internal/fileutils"
+
+	"github.com/ActiveState/cli/internal/osutils"
+	"github.com/ActiveState/cli/internal/output"
+	"github.com/ActiveState/cli/internal/subshell"
+	"github.com/ActiveState/cli/internal/subshell/sscommon"
+)
+
+// NOTE: From: internal/runners/clean/uninstall.go
+
+func runUnInstall(out output.Outputer, params *Params) error {
+	// This is the deploy id constant: DeployID RcIdentification = RcIdentification{
+	installToDir := params.path
+	// Should look for license file to see if it exists before we delete the install directory
+	uninstallFileCheck := filepath.Join(installToDir, "LICENSE.txt")
+
+	if !fileutils.FileExists(uninstallFileCheck) {
+		return errs.New("Doesn't seem like given directory is an install directory")
```

Comment:
Change to something like this:
`ActiveState license not found. Please specify a valid install directory.`

Reply:
Though, if the process deletes the LICENSE.txt, and hasn't completed the user will be stuck.  Probably we'll need the user to be allowed to proceed with file removal anyway.  `Are you sure you want to uninstall ? y/[N]`

Reply:
Okay, changed the behaviour.